db/schema.rb

```diff
@@ -427,6 +427,19 @@
     t.index ["legal_aid_application_id"], name: "index_dependants_on_legal_aid_application_id"
   end
 
+  create_table "disregards", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
should we not be calling this policy_disregards to avoid confusion with benefit disregards?

Reply:
or perhaps `capital_disregards` (if it makes business sense) if we are avoiding the `policy_regards` name due to the other table

Reply:
Hmm, yeah probably, we already have an existing PolicyDisregards table though which we were going to leave for a while until we migrate the data. Perhaps we'd need to rename this to `LegacyPolicyDisregards`? Or alternatively migrate all the old policy disregards data in this PR. 

@kmahern and @skdejong what do you guys think?

Reply:
I think `capital_disregards` would make the most sense out of all those options.

Reply:
I think I agree with @skdejong that capital_disregards might be the best option. For consistency should we also make the namespace capital_disregards?

Reply:
Yes, I think we should keep things consistent.